### PR TITLE
Update and refresh Publish packages

### PR DESCRIPTION
**Pull request by v-thepet**
User Story https://dev.azure.com/mseng/TechnicalContent/_workitems/edit/1973816

### CODE REVIEW

**Comment by v-thepet** [comment]
@Mikejo5000 @nkolev92 @zivkan @JonDouglas - This article is ready for your review. The article has a couple of includes that were edited in PR.https://github.com/NuGet/docs.microsoft.com-nuget/pull/2862. Please review the includes in that PR. Once that PR is merged, I'll uncomment the includes calls in this article. Thanks for your timely review, but please don't merge this PR until the other PR is merged and the includes calls in this one are uncommented.

Note the warnings caused by invalid author/ms.author in two files that aren't changed in this PR. I started getting these warnings in all my open PRs as of last night 8/29. The author/ms.author are invalid in the upstream, but several other files in the repo also have these invalid names, and aren't being pulled in to PRs. I can't explain it. Please review this article and I'll work on getting the extra files out of the PRs. Thanks.

**Comment by v-thepet** [comment]
@Mikejo5000 @nkolev92 @zivkan @JonDouglas - I changed my mind and uncommented the includes. I didn't want to risk this getting merged with them commented out. When reviewing the sections **Create an API key** and **Use the dotnet CLI**, please make any edits in the edited includes at PR #2862. Thanks.

**Review by zivkan — APPROVED** [review]

**Comment by zivkan** [comment]
Since this docs page is in the `nuget-org` directory, I'll wait a day to see if anyone from the server team wants to review.

**Review by joelverhagen — APPROVED** [review]

File: docs/nuget-org/Publish-a-package.md [hunk 1]
```diff
@@ -1,122 +1,125 @@
...
-
-This article covers publishing to nuget.org; for publishing to Azure Artifacts, see [Package Management](https://www.visualstudio.com/docs/package/nuget/publish).
+You can also make private packages available to only a team or organization by hosting them on a file share, a private NuGet server, or a third-party repository such as myget, ProGet, Nexus Repository, or Artifactory. For more information, see [Host your own NuGet feeds](../hosting-packages/overview.md). For publishing with [Azure Artifacts](https://www.visualstudio.com/docs/package/nuget/publish), see [Publish packages to NuGet.org](/azure/devops/artifacts/nuget/publish-to-nuget-org).
 
 ## Publish to nuget.org
 
-For nuget.org, you must sign in with a Microsoft account, with which you'll be asked to register the account with nuget.org.
+To publish on nuget.org, sign in to nuget.org with a Microsoft account, and use it to create a free nuget.org account. Follow the instructions at [Add a new individual account](individual-accounts.md#add-a-new-individual-account).
+
+![Screenshot that shows the NuGet sign in link.](media/publish_NuGetSignIn.png)
```
Comment (zhhyu) [inline docs/nuget-org/Publish-a-package.md:21]: media/publish_NuGetSignIn.png -> media/publish_NuGetSignIn.PNG?

Seems that the image path is broken.

**Review by zhhyu — APPROVED** [review]

File: docs/nuget-org/Publish-a-package.md (hunk 1 above)
Comment (v-thepet) [reply docs/nuget-org/Publish-a-package.md]: @zhhyu - It works for me, but the uppercase extension is nonstandard and has caused problems in other repos so I changed it (for both graphics). (Usually the only way to do this from Windows machines is to rename the graphics.)

**Comment by v-thepet** [comment]
@zivkan @nkolev92 - This should be ready to merge, thanks.

**Pull request merged**